### ISSUE TITLE
Evolution for the next 5 hours is empty

### ISSUE THREAD

**Issue by DocHifi** [atareao/my-weather-indicator#132]
Evolution for the next 5 hours is empty, only shows a white window.
Ubuntu 22.04 and Debian Bookworm, both with Mate Desktop on different laptops.

**Comment by pad0815** [comment]
The same on me on (Ubuntu 22.04) - only a blank windows appear "The evolution of the temperature and pressure" is not working.

**Comment by DocHifi** [comment]
7 month later, no fix.
Looks like the project is dead, no reaction.

**Comment by pad0815** [comment]
Yes, but a small drop of bitterness .. on my old PC with Ubuntu 20.04 with the provider Open Meteo, the evolution is working.
